### ISSUE TITLE
Broken Integration Tests

### ISSUE THREAD

**Issue by danehans** [kubernetes-sigs/kubefed#226]
By following the [dev guide](https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/development.md#running-tests), I'm unable to have integration tests pass against master with the following error:
```
--- FAIL: TestIntegration/Parallel-Integration-Test-Group/TestCrud/FederatedSecret (30.32s)
	logger.go:52: Creating new FederatedSecret in namespace "crud-federatedsecret-6hmjm"
	logger.go:52: Created new FederatedSecret "crud-federatedsecret-6hmjm/test-crud-6p2kq"
   	logger.go:52: Creating new FederatedSecretPlacement in namespace "crud-federatedsecret-6hmjm"
   	logger.go:52: Created new FederatedSecretPlacement "crud-federatedsecret-6hmjm/test-crud-6p2kq"
   	logger.go:52: Creating new FederatedSecretOverride in namespace "crud-federatedsecret-6hmjm"
    	logger.go:52: Created new FederatedSecretOverride "crud-federatedsecret-6hmjm/test-crud-6p2kq"
   	logger.go:48: Resource versions for crud-federatedsecret-6hmjm/test-crud-6p2kq: template "130", placement "131", override "133"
  	logger.go:44: Error waiting for propagated version for Secret "crud-federatedsecret-6hmjm/test-crud-6p2kq": timed out waiting for the condition
```

See [here](https://gist.githubusercontent.com/danehans/0a7ccafb6d9c66c11106912895684599/raw/43f775abd77cc1a906ebc1fbea3616e77251a6b2/integration_test_fail) for full log output.

**Comment by marun** [comment]
@danehans The tests are passing in CI, which suggest the problem is your environment setup.  Please provide details as to what component versions you are experiencing the problem with.

**Comment by danehans** [comment]
@marun here are more details about my issue running integration tests:

System Details:
```
$ lsb_release --all
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.5 LTS
Release:	16.04
Codename:	xenial
```
Integration Test Prerequisites:
```
$ kubebuilder version
Version: version.Version{KubeBuilderVersion:"1.0.0", KubernetesVendor:"1.10", GitCommit:"9c4c6c213a8d17f8c21cf4f1aa9cefb99fbbf5ca", BuildDate:"2018-07-19T23:21:03Z", GoOs:"unknown", GoArch:"unknown"}

$ etcd --version
etcd Version: 3.3.0+git
Git SHA: Not provided (use ./build instead of go build)
Go Version: go1.10.1
Go OS/Arch: linux/amd64

$ kube-apiserver --version
Kubernetes v1.11.0

$ which kubebuilder
/usr/local/kubebuilder/bin/kubebuilder
$ which etcd
/usr/local/kubebuilder/bin/etcd
$ which kube-apiserver
/usr/local/bin/kube-apiserver

$ export TEST_ASSET_PATH="/usr/local/kubebuilder/bin"
$ export TEST_ASSET_ETCD="${TEST_ASSET_PATH}/etcd"
$ export TEST_ASSET_KUBE_APISERVER="${TEST_ASSET_PATH}/kube-apiserver"
```

I run a test for one of the supported types and it fails:
```
cd test/integration &&
> go test -v -run ^TestIntegration/Parallel-Integration-Test-Group/TestCrud/FederatedSecret$
=== RUN   TestIntegration
Flag --insecure-port has been deprecated, This flag will be removed in a future version.
Flag --insecure-bind-address has been deprecated, This flag will be removed in a future version.
I0824 18:13:55.246064   13470 server.go:135] Version: v0.0.0-master+$Format:%h$
I0824 18:13:55.972242   13470 serving.go:275] Generated self-signed cert (/tmp/k8s_test_framework_944912900/apiserver.crt, /tmp/k8s_test_framework_944912900/apiserver.key)
I0824 18:13:55.972316   13470 server.go:721] external host was not specified, using 10.138.0.2
W0824 18:13:55.972329   13470 authentication.go:377] AnonymousAuth is not allowed with the AllowAll authorizer.  Resetting AnonymousAuth to false. You should use a different authorizer
W0824 18:13:56.062094   13470 admission.go:68] PersistentVolumeLabel admission controller is deprecated. Please remove this controller from your configuration files and scripts.
I0824 18:13:56.074837   13470 plugins.go:149] Loaded 9 admission controller(s) successfully in the following order: NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultTolerationSeconds,PersistentVolumeLabel,DefaultStorageClass,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota.
W0824 18:13:56.075420   13470 admission.go:68] PersistentVolumeLabel admission controller is deprecated. Please remove this controller from your configuration files and scripts.
I0824 18:13:56.075895   13470 plugins.go:149] Loaded 9 admission controller(s) successfully in the following order: NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultTolerationSeconds,PersistentVolumeLabel,DefaultStorageClass,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota.
I0824 18:13:56.121720   13470 master.go:228] Using reconciler: master-count
W0824 18:13:56.286955   13470 genericapiserver.go:342] Skipping API batch/v2alpha1 because it has no resources.
W0824 18:13:56.308795   13470 genericapiserver.go:342] Skipping API rbac.authorization.k8s.io/v1alpha1 because it has no resources.
W0824 18:13:56.310606   13470 genericapiserver.go:342] Skipping API storage.k8s.io/v1alpha1 because it has no resources.
W0824 18:13:56.325435   13470 genericapiserver.go:342] Skipping API admissionregistration.k8s.io/v1alpha1 because it has no resources.
[restful] 2018/08/24 18:13:56 log.go:33: [restful/swagger] listing is available at http://10.138.0.2:443/swaggerapi
[restful] 2018/08/24 18:13:56 log.go:33: [restful/swagger] http://10.138.0.2:443/swaggerui/ is mapped to folder /swagger-ui/
[restful] 2018/08/24 18:13:57 log.go:33: [restful/swagger] listing is available at http://10.138.0.2:443/swaggerapi
[restful] 2018/08/24 18:13:57 log.go:33: [restful/swagger] http://10.138.0.2:443/swaggerui/ is mapped to folder /swagger-ui/
W0824 18:13:58.043025   13470 admission.go:68] PersistentVolumeLabel admission controller is deprecated. Please remove this controller from your configuration files and scripts.
I0824 18:13:58.043575   13470 plugins.go:149] Loaded 9 admission controller(s) successfully in the following order: NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultTolerationSeconds,PersistentVolumeLabel,DefaultStorageClass,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota.
I0824 18:14:01.331651   13470 insecure_handler.go:121] Serving insecurely on 127.0.0.1:46175
I0824 18:14:01.333086   13470 available_controller.go:262] Starting AvailableConditionController
I0824 18:14:01.333254   13470 cache.go:32] Waiting for caches to sync for AvailableConditionController controller
I0824 18:14:01.335726   13470 controller.go:84] Starting OpenAPI AggregationController
I0824 18:14:01.337280   13470 crdregistration_controller.go:110] Starting crd-autoregister controller
I0824 18:14:01.350801   13470 controller_utils.go:1019] Waiting for caches to sync for crd-autoregister controller
I0824 18:14:01.355147   13470 crd_finalizer.go:242] Starting CRDFinalizer
I0824 18:14:01.355672   13470 apiservice_controller.go:90] Starting APIServiceRegistrationController
I0824 18:14:01.355693   13470 cache.go:32] Waiting for caches to sync for APIServiceRegistrationController controller
I0824 18:14:01.358793   13470 customresource_discovery_controller.go:174] Starting DiscoveryController
I0824 18:14:01.358828   13470 naming_controller.go:276] Starting NamingConditionController
I0824 18:14:01.433830   13470 cache.go:39] Caches are synced for AvailableConditionController controller
I0824 18:14:01.451051   13470 controller_utils.go:1026] Caches are synced for crd-autoregister controller
I0824 18:14:01.451142   13470 autoregister_controller.go:136] Starting autoregister controller
I0824 18:14:01.451152   13470 cache.go:32] Waiting for caches to sync for autoregister controller
I0824 18:14:01.455831   13470 cache.go:39] Caches are synced for APIServiceRegistrationController controller
I0824 18:14:01.470264   13470 controller.go:537] quota admission added evaluator for: { endpoints}
I0824 18:14:01.551301   13470 cache.go:39] Caches are synced for autoregister controller
2018/08/24 18:14:01 Creating CRD clusters.clusterregistry.k8s.io
2018/08/24 18:14:01 Finished installing.
2018/08/24 18:14:01 Creating CRD federatedclusters.core.federation.k8s.io
2018/08/24 18:14:01 Creating CRD federatedconfigmaps.core.federation.k8s.io
2018/08/24 18:14:01 Creating CRD federatedconfigmapoverrides.core.federation.k8s.io
2018/08/24 18:14:01 Creating CRD federatedconfigmapplacements.core.federation.k8s.io
2018/08/24 18:14:01 Creating CRD federateddeployments.core.federation.k8s.io
2018/08/24 18:14:02 Creating CRD federateddeploymentoverrides.core.federation.k8s.io
2018/08/24 18:14:02 Creating CRD federateddeploymentplacements.core.federation.k8s.io
2018/08/24 18:14:03 Creating CRD federatedingresses.core.federation.k8s.io
2018/08/24 18:14:03 Creating CRD federatedingressplacements.core.federation.k8s.io
2018/08/24 18:14:03 Creating CRD federatedjobs.core.federation.k8s.io
2018/08/24 18:14:04 Creating CRD federatedjoboverrides.core.federation.k8s.io
2018/08/24 18:14:04 Creating CRD federatedjobplacements.core.federation.k8s.io
2018/08/24 18:14:05 Creating CRD federatednamespaceplacements.core.federation.k8s.io
2018/08/24 18:14:05 Creating CRD federatedreplicasets.core.federation.k8s.io
2018/08/24 18:14:05 Creating CRD federatedreplicasetoverrides.core.federation.k8s.io
2018/08/24 18:14:06 Creating CRD federatedreplicasetplacements.core.federation.k8s.io
2018/08/24 18:14:06 Creating CRD federatedsecrets.core.federation.k8s.io
2018/08/24 18:14:07 Creating CRD federatedsecretoverrides.core.federation.k8s.io
2018/08/24 18:14:07 Creating CRD federatedsecretplacements.core.federation.k8s.io
2018/08/24 18:14:07 Creating CRD federatedservices.core.federation.k8s.io
2018/08/24 18:14:08 Creating CRD federatedserviceplacements.core.federation.k8s.io
2018/08/24 18:14:08 Creating CRD federatedtypeconfigs.core.federation.k8s.io
2018/08/24 18:14:09 Creating CRD propagatedversions.core.federation.k8s.io
2018/08/24 18:14:09 Creating CRD dnsendpoints.multiclusterdns.federation.k8s.io
2018/08/24 18:14:09 Creating CRD multiclusterservicednsrecords.multiclusterdns.federation.k8s.io
2018/08/24 18:14:10 Creating CRD replicaschedulingpreferences.scheduling.federation.k8s.io
2018/08/24 18:14:10 Finished installing.
I0824 18:14:10.565341   13470 controller.go:537] quota admission added evaluator for: {clusterregistry.k8s.io clusters}
I0824 18:14:10.577367   13470 controller.go:537] quota admission added evaluator for: {core.federation.k8s.io federatedclusters}
Flag --insecure-port has been deprecated, This flag will be removed in a future version.
Flag --insecure-bind-address has been deprecated, This flag will be removed in a future version.
I0824 18:14:10.743493   13482 server.go:135] Version: v0.0.0-master+$Format:%h$
I0824 18:14:11.164668   13482 serving.go:275] Generated self-signed cert (/tmp/k8s_test_framework_458614163/apiserver.crt, /tmp/k8s_test_framework_458614163/apiserver.key)
I0824 18:14:11.164701   13482 server.go:721] external host was not specified, using 10.138.0.2
W0824 18:14:11.164713   13482 authentication.go:377] AnonymousAuth is not allowed with the AllowAll authorizer.  Resetting AnonymousAuth to false. You should use a different authorizer
W0824 18:14:11.167214   13482 admission.go:68] PersistentVolumeLabel admission controller is deprecated. Please remove this controller from your configuration files and scripts.
I0824 18:14:11.167692   13482 plugins.go:149] Loaded 9 admission controller(s) successfully in the following order: NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultTolerationSeconds,PersistentVolumeLabel,DefaultStorageClass,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota.
W0824 18:14:11.168109   13482 admission.go:68] PersistentVolumeLabel admission controller is deprecated. Please remove this controller from your configuration files and scripts.
I0824 18:14:11.168489   13482 plugins.go:149] Loaded 9 admission controller(s) successfully in the following order: NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultTolerationSeconds,PersistentVolumeLabel,DefaultStorageClass,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota.
I0824 18:14:11.175638   13482 master.go:228] Using reconciler: master-count
W0824 18:14:11.269634   13482 genericapiserver.go:342] Skipping API batch/v2alpha1 because it has no resources.
W0824 18:14:11.294340   13482 genericapiserver.go:342] Skipping API rbac.authorization.k8s.io/v1alpha1 because it has no resources.
W0824 18:14:11.298842   13482 genericapiserver.go:342] Skipping API storage.k8s.io/v1alpha1 because it has no resources.
W0824 18:14:11.336018   13482 genericapiserver.go:342] Skipping API admissionregistration.k8s.io/v1alpha1 because it has no resources.
[restful] 2018/08/24 18:14:11 log.go:33: [restful/swagger] listing is available at http://10.138.0.2:443/swaggerapi
[restful] 2018/08/24 18:14:11 log.go:33: [restful/swagger] http://10.138.0.2:443/swaggerui/ is mapped to folder /swagger-ui/
[restful] 2018/08/24 18:14:12 log.go:33: [restful/swagger] listing is available at http://10.138.0.2:443/swaggerapi
[restful] 2018/08/24 18:14:12 log.go:33: [restful/swagger] http://10.138.0.2:443/swaggerui/ is mapped to folder /swagger-ui/
W0824 18:14:12.962647   13482 admission.go:68] PersistentVolumeLabel admission controller is deprecated. Please remove this controller from your configuration files and scripts.
I0824 18:14:12.963155   13482 plugins.go:149] Loaded 9 admission controller(s) successfully in the following order: NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultTolerationSeconds,PersistentVolumeLabel,DefaultStorageClass,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota.
I0824 18:14:16.267880   13482 insecure_handler.go:121] Serving insecurely on 127.0.0.1:33579
I0824 18:14:16.270441   13482 apiservice_controller.go:90] Starting APIServiceRegistrationController
I0824 18:14:16.270462   13482 cache.go:32] Waiting for caches to sync for APIServiceRegistrationController controller
I0824 18:14:16.270487   13482 controller.go:84] Starting OpenAPI AggregationController
I0824 18:14:16.270649   13482 crd_finalizer.go:242] Starting CRDFinalizer
I0824 18:14:16.270675   13482 available_controller.go:262] Starting AvailableConditionController
I0824 18:14:16.270683   13482 cache.go:32] Waiting for caches to sync for AvailableConditionController controller
I0824 18:14:16.270706   13482 crdregistration_controller.go:110] Starting crd-autoregister controller
I0824 18:14:16.270714   13482 controller_utils.go:1019] Waiting for caches to sync for crd-autoregister controller
I0824 18:14:16.272160   13482 customresource_discovery_controller.go:174] Starting DiscoveryController
I0824 18:14:16.272183   13482 naming_controller.go:276] Starting NamingConditionController
I0824 18:14:16.298170   13482 controller.go:537] quota admission added evaluator for: { endpoints}
I0824 18:14:16.370863   13482 controller_utils.go:1026] Caches are synced for crd-autoregister controller
I0824 18:14:16.370878   13482 cache.go:39] Caches are synced for AvailableConditionController controller
I0824 18:14:16.371031   13482 autoregister_controller.go:136] Starting autoregister controller
I0824 18:14:16.371055   13482 cache.go:32] Waiting for caches to sync for autoregister controller
I0824 18:14:16.374981   13482 cache.go:39] Caches are synced for APIServiceRegistrationController controller
I0824 18:14:16.471231   13482 cache.go:39] Caches are synced for autoregister controller
=== RUN   TestIntegration/Parallel-Integration-Test-Group
=== RUN   TestIntegration/Parallel-Integration-Test-Group/TestCrud
=== PAUSE TestIntegration/Parallel-Integration-Test-Group/TestCrud
=== CONT  TestIntegration/Parallel-Integration-Test-Group/TestCrud
=== RUN   TestIntegration/Parallel-Integration-Test-Group/TestCrud/FederatedSecret
E0824 18:14:16.761575   13451 federated_informer.go:213] Cluster &{{FederatedCluster core.federation.k8s.io/v1alpha1} {test-cluster-lgsg4  federation-system-gscx4 /apis/core.federation.k8s.io/v1alpha1/namespaces/federation-system-gscx4/federatedclusters/test-cluster-lgsg4 7f9895cc-a7c9-11e8-83cc-42010a8a0002 94 1 2018-08-24 18:14:10 +0000 UTC <nil> <nil> map[] map[] [] nil [] } {{test-cluster-lgsg4} &LocalObjectReference{Name:test-cluster-lgsg4-credentialsbwck5,}} {[]  }} not added.  Not of correct type, or cluster not ready.
E0824 18:14:16.762692   13451 federated_informer.go:213] Cluster &{{FederatedCluster core.federation.k8s.io/v1alpha1} {test-cluster-nlvsf  federation-system-gscx4 /apis/core.federation.k8s.io/v1alpha1/namespaces/federation-system-gscx4/federatedclusters/test-cluster-nlvsf 83330108-a7c9-11e8-83cc-42010a8a0002 128 1 2018-08-24 18:14:16 +0000 UTC <nil> <nil> map[] map[] [] nil [] } {{test-cluster-nlvsf} &LocalObjectReference{Name:test-cluster-nlvsf-credentialsf4lwc,}} {[]  }} not added.  Not of correct type, or cluster not ready.
E0824 18:14:16.764998   13451 federated_informer.go:213] Cluster &{{FederatedCluster core.federation.k8s.io/v1alpha1} {test-cluster-lgsg4  federation-system-gscx4 /apis/core.federation.k8s.io/v1alpha1/namespaces/federation-system-gscx4/federatedclusters/test-cluster-lgsg4 7f9895cc-a7c9-11e8-83cc-42010a8a0002 94 1 2018-08-24 18:14:10 +0000 UTC <nil> <nil> map[] map[] [] nil [] } {{test-cluster-lgsg4} &LocalObjectReference{Name:test-cluster-lgsg4-credentialsbwck5,}} {[]  }} not added.  Not of correct type, or cluster not ready.
E0824 18:14:16.765788   13451 federated_informer.go:213] Cluster &{{FederatedCluster core.federation.k8s.io/v1alpha1} {test-cluster-nlvsf  federation-system-gscx4 /apis/core.federation.k8s.io/v1alpha1/namespaces/federation-system-gscx4/federatedclusters/test-cluster-nlvsf 83330108-a7c9-11e8-83cc-42010a8a0002 128 1 2018-08-24 18:14:16 +0000 UTC <nil> <nil> map[] map[] [] nil [] } {{test-cluster-nlvsf} &LocalObjectReference{Name:test-cluster-nlvsf-credentialsf4lwc,}} {[]  }} not added.  Not of correct type, or cluster not ready.
I0824 18:14:16.767181   13470 controller.go:537] quota admission added evaluator for: { namespaces}
I0824 18:14:16.828146   13470 controller.go:537] quota admission added evaluator for: {core.federation.k8s.io federatedsecrets}
I0824 18:14:16.832452   13470 controller.go:537] quota admission added evaluator for: {core.federation.k8s.io federatedsecretplacements}
I0824 18:14:16.846077   13470 controller.go:537] quota admission added evaluator for: {core.federation.k8s.io federatedsecretoverrides}
E0824 18:14:16.853002   13451 controller.go:463] Failed to ensure finalizers for FederatedSecret "crud-federatedsecret-b6msk/test-crud-spnhl": failed to add finalizers map[orphan:{} federation.kubernetes.io/delete-from-underlying-clusters:{}] to object crud-federatedsecret-b6msk/test-crud-spnhl: Operation cannot be fulfilled on federatedsecrets.core.federation.k8s.io "test-crud-spnhl": the object has been modified; please apply your changes to the latest version and try again
I0824 18:14:47.350678   13482 crdregistration_controller.go:139] Shutting down crd-autoregister controller
I0824 18:14:47.350845   13482 autoregister_controller.go:160] Shutting down autoregister controller
I0824 18:14:47.350946   13482 apiservice_controller.go:102] Shutting down APIServiceRegistrationController
I0824 18:14:47.350968   13482 customresource_discovery_controller.go:185] Shutting down DiscoveryController
I0824 18:14:47.350985   13482 naming_controller.go:287] Shutting down NamingConditionController
I0824 18:14:47.351002   13482 available_controller.go:274] Shutting down AvailableConditionController
I0824 18:14:47.351037   13482 crd_finalizer.go:254] Shutting down CRDFinalizer
I0824 18:14:47.351163   13482 controller.go:90] Shutting down OpenAPI AggregationController
I0824 18:14:47.351487   13482 serve.go:136] Stopped listening on 127.0.0.1:33579
I0824 18:14:47.478522   13470 customresource_discovery_controller.go:185] Shutting down DiscoveryController
I0824 18:14:47.478556   13470 naming_controller.go:287] Shutting down NamingConditionController
I0824 18:14:47.478574   13470 apiservice_controller.go:102] Shutting down APIServiceRegistrationController
I0824 18:14:47.478588   13470 crd_finalizer.go:254] Shutting down CRDFinalizer
I0824 18:14:47.478604   13470 crdregistration_controller.go:139] Shutting down crd-autoregister controller
I0824 18:14:47.478626   13470 available_controller.go:274] Shutting down AvailableConditionController
I0824 18:14:47.479007   13470 controller.go:90] Shutting down OpenAPI AggregationController
I0824 18:14:47.479141   13470 autoregister_controller.go:160] Shutting down autoregister controller
I0824 18:14:47.479243   13470 serve.go:136] Stopped listening on 127.0.0.1:46175
--- FAIL: TestIntegration (56.26s)
	logger.go:52: Starting a federation of 2 clusters...
	logger.go:52: Creating Cluster Registry CRD
	logger.go:52: Creating Federation CRDs
	logger.go:52: Added cluster test-cluster-lgsg4 to the federation
	logger.go:52: Added cluster test-cluster-nlvsf to the federation
	logger.go:52: Starting cluster controller
	logger.go:48: Federation started.
    --- FAIL: TestIntegration/Parallel-Integration-Test-Group (0.00s)
        --- FAIL: TestIntegration/Parallel-Integration-Test-Group/TestCrud (30.50s)
            --- FAIL: TestIntegration/Parallel-Integration-Test-Group/TestCrud/FederatedSecret (30.49s)
            	logger.go:52: Creating new FederatedSecret in namespace "crud-federatedsecret-b6msk"
            	logger.go:52: Created new FederatedSecret "crud-federatedsecret-b6msk/test-crud-spnhl"
            	logger.go:52: Creating new FederatedSecretPlacement in namespace "crud-federatedsecret-b6msk"
            	logger.go:52: Created new FederatedSecretPlacement "crud-federatedsecret-b6msk/test-crud-spnhl"
            	logger.go:52: Creating new FederatedSecretOverride in namespace "crud-federatedsecret-b6msk"
            	logger.go:52: Created new FederatedSecretOverride "crud-federatedsecret-b6msk/test-crud-spnhl"
            	logger.go:48: Resource versions for crud-federatedsecret-b6msk/test-crud-spnhl: template "131", placement "132", override "133"
            	logger.go:44: Error waiting for propagated version for Secret "crud-federatedsecret-b6msk/test-crud-spnhl": timed out waiting for the condition
FAIL
exit status 1
FAIL	github.com/kubernetes-sigs/federation-v2/test/integration	56.275s
```

**Comment by font** [comment]
@danehans Try to blow away your `./bin` directory and re-run `./scripts/download-binaries.sh` to get the latest versions.
